README.md

```diff
@@ -1,72 +1,97 @@
+## Requesting access to CNCF repositories using CLOWarden and config.yaml
+
+The CNCF has centralized access control for repos in this org using a application called CLOWarden [src](https://github.com/cncf/clowarden) [site](https://clowarden.io/audit/).
+
+To request access to a repository
 
-## Listing Formats
+- first invite the user to join the CNCF org
+- then submit a pull request to modify [config.yaml](./config.yaml) file as described below
 
-### config.yaml configures CNCF org repository access
+Your submitted changes will be checked and validated on the Pull Request by CLOWarden.
 
-Use config.yaml here to control repo-level access to your CNCF org repository.
+If there are problems with your changes, you will receive clear instructions from CLOWarden on how to fix them.
 
-config.yaml documents the people, and teams of people, who are granted access to GitHub repositories in the CNCF org.
+The access rules contained in config.yaml are the single source of truth that document who has access to repos in the CNCF Org on GitHub.
+After each merged pull request config.yaml is read and processed automatically by CLOWarden to grant the stated access.
 
-DO NOT USE settings.yml at the repo-level to control permission; if you do, [Sheriff](#notes) will overwite permissions described in the repo-level settings.yml with the settings described here in config.yaml.
+In the config.yaml file there are two ways to grant repo access to a community member
 
-A person's GitHub profile is used to grant access to a repository or define membership of a team.
+- add their GitHub username to a repository entry
+  
+  OR
 
-Adding an entry to repositories allow you to describe who has access to your repoistory.
+- add them to a team and then add the team to a repository entry
 
 ```yaml
 repositories:
-  - name: repo_name
+  - name: REPO_NAME
     external_collaborators:
-      github_profile_1: read | triage | write | maintain | admin
-              |
-      github_profile_n: read | triage | write | maintain | admin
+      GITHUB_USERNAME_1: read | triage | write | maintain | admin
+              :
+              :
+      GITHUB_USERNAME_N: read | triage | write | maintain | admin
     teams:
-      team_name_1: read | triage | write | maintain | admin
-              |
-      team_name_n: read | triage | write | maintain | admin
+      TEAM_NAME_1: read | triage | write | maintain | admin
+              :
+      TEAM_NAME_N: read | triage | write | maintain | admin
     # Optional repository settings
     settings:
       has_wiki: true|false # Default is false
     visibility: public | private # Default is public
 ```
 
-Note: the ```name: repo name``` does not necessarily appear as the first field in a repositories entry which can be confusing.
+> [!IMPORTANT]
+>
+> - Invite users to join the CNCF Org *before you add them via a PR* to config.yaml
```

Comment:
Same as above. Actually, this doesn't apply to the `repositories` section, only to the `teams` top level section (please see sample in my other comment).